lib/matplotlib/tests/test_figure.py

```diff
@@ -382,9 +383,8 @@ def test_subplots_shareax_loglabels():
 
 def test_savefig():
     fig = plt.figure()
-    msg = r"savefig\(\) takes 2 positional arguments but 3 were given"
-    with pytest.raises(TypeError, match=msg):
```

Comment:
I'm not sure that these replacements are beneficial to the tests. Understanding bind is more difficult and less readable.